server/src/main/java/org/elasticsearch/action/admin/indices/recovery/RecoveryRequest.java

```diff
@@ -31,7 +31,7 @@
  */
 public class RecoveryRequest extends BroadcastRequest<RecoveryRequest> {
 
-    private boolean detailed = false;       // Provides extra details in the response
+    private boolean details = false;       // Provides extra details in the response
```

Comment:
can we remove this too?

Reply:
The `details` variable in `RecoveryRequest` ? It's to pass the `details` flag in Recovery API ? can we remove it ?